src/Places/DataAccess.ts

```diff
@@ -134,15 +135,26 @@ export async function getPlaceMedia(id: string): Promise<Medium[]> {
 }
 
 async function getFromApi(toBeFetchedFromAPI: string[]): Promise<any> {
-	const apiResponse = await StApi.get('places?' + stringify({
-		ids: toBeFetchedFromAPI.join('|')
+	const CHUNK_SIZE: number = 50;
```

Comment:
I see you explicitly define types when not needed,  but if it's intentional, okok...